dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -893,3 +908,44 @@ ERROR_INVALID_FEATURE_BOM_COORDINATE.explanation=The features BOM Maven coordina
 ERROR_INVALID_FEATURE_BOM_COORDINATE.useraction=Reformat the features BOM Maven coordinate to fit the groupId:artifactId:version format.
 
 MSG_STABILIZING_FEATUREMANAGER=This featureManager action is stabilized. Use installUtility {0} instead. The installUtility command is recommended for installation and repository-related actions.
+
+ERROR_VERIFY_OPTION_DOES_NOT_MATCH=CWWKF1504E: The feature verify option "{0}" does not match "{1}" in the command line.
+ERROR_VERIFY_OPTION_DOES_NOT_MATCH.explanation=The feature verify option on properties file or environment variable should match the one in the command line. 
+ERROR_VERIFY_OPTION_DOES_NOT_MATCH.useraction=Ensure specified feature verify options match each other or choose one option.
+
+ERROR_VERIFY_OPTION_NOT_VALID=CWWKF1505E: The verify option {0} isn't valid. Ensure that the option is valid.
+ERROR_VERIFY_OPTION_NOT_VALID.explanation=The specified option is not valid. 
+ERROR_VERIFY_OPTION_NOT_VALID.useraction=Specify the valid option from the following: {"enforce", "warn", "skip", "all"}.
+
+ERROR_FAILED_TO_DOWNLOAD_KEY_FROM_KEY_URL=CWWKF1506E: The public key could not be downloaded: {0}.
+ERROR_FAILED_TO_DOWNLOAD_KEY_FROM_KEY_URL.explanation=The specified public key was not downloaded from the specified key URL. 
+ERROR_FAILED_TO_DOWNLOAD_KEY_FROM_KEY_URL.useraction=Verify your key ID, key URL and network access, then try the action again. If the connection still fails, the key URL might be temporarily unavailable.
+
+ERROR_KEYID_NOT_PROVIDED=CWWKF1508E: The public key ID for {0} key URL was not provided. 
+ERROR_KEYID_NOT_PROVIDED.explanation=The public key ID for the specified key URL was not provided. The 64 bits key ID is required to download and verify the authenticity of the key. 
+ERROR_KEYID_NOT_PROVIDED.useraction=Provide the key ID in long (64 bits) format. To find out the key ID, run GnuPG command. 
+
+ERROR_KEYURL_UNSUPPORTED_PROTOCOL=CWWKF1509E: The URL protocol for the following key URL is not supported: {0}. Supported protocols are HTTP, HTTPS, and file. Ensure that the URL is specified correctly.
+ERROR_KEYURL_UNSUPPORTED_PROTOCOL.explanation=The URL protocol for the specified key URL is not supported.
+ERROR_KEYURL_UNSUPPORTED_PROTOCOL.useraction=Ensure that the URL is specified correctly. Supported protocols are HTTP, HTTPS, and file. 
+
+ERROR_REVOKED_PUBLIC_KEY=CWWKF1510E: The public key ID {0} has been revoked. 
+ERROR_REVOKED_PUBLIC_KEY.explanation=The public key has been revoked. 
+ERROR_REVOKED_PUBLIC_KEY.useraction=Provide the new public key ID and signature(s). If the revoked key is Liberty public key, apply the interim fix or provide the new public key ID available on IBM website.
+
+ERROR_EXPIRED_PUBLIC_KEY=CWWKF1511E: The public key ID {0} has been expired on {1}.
+ERROR_EXPIRED_PUBLIC_KEY.explanation=The public key has been expired.
```

Comment:
```suggestion
ERROR_EXPIRED_PUBLIC_KEY.explanation=The public key is expired.
```

same here, provide more context if possible. Not sure what that would be in this case though